CMakeLists.txt

```diff
@@ -1410,13 +1412,13 @@ if (NOT EVENT__DISABLE_TESTS)
         add_custom_target(verify COMMAND "${VERIFY_PATH}"
                           DEPENDS event ${ALL_TESTPROGS})
     else()
-        # On some platforms doing exec(unset) as CMake does won't work, so make sure
+        # On some platforms doing exec(unset) as CMake doesn't work, so make sure
         # we run the unset command in a shell instead.
         # First we write the script contents.
         file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/tmp/verify_tests.sh
             "
             #!/bin/bash
-            unset EVENT_NOEPOLL; unset EVENT_NOPOLL; unset EVENT_NOSELECT; unset EVENT_NOWIN32; unset EVENT_NOEVPORT; unset EVENT_NOKQUEUE; unset EVENT_NODEVPOLL
+            unset EVENT_NOEPOLL; unset EVENT_NOPOLL; unset EVENT_NOSELECT; unset EVENT_NOWIN32; unset EVENT_NOEVPORT; unset EVENT_NOKQUEUE; unset EVENT_NODEVPOLL; unset EVENT_NOWEPOLL
```

Comment:
Am I understand correctly that wepoll do not support cygwin (and related stuff), hence we do not need to modify [test.sh](https://github.com/libevent/libevent/blob/f0b3160f8ce7fbd411493dcd023f562f4f9d17ee/test/test.sh#L3) ?

Reply:
I have limited experience with cygwin so my initial answer is, idk. I did not actively look into supporting the wepoll backend on cygwin. Is win32 backend supported on cygwin? My initial reading of this line was that the "unset EVENT_NOWIN32" was not needed so I added EVENT_NOWEPOLL strictly for consistency. Should revert this change?

Reply:
>I have limited experience with cygwin so my initial answer is

Me too (and I planning to keep it that way) :)

>Should I revert this change?

Yes I guess it can be dropped, but this is not significant